src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -479,10 +526,12 @@ private PSObject[] InvokePowerShell(string command, IDictionary<string, object>
                     powerShell.AddParameter(kvp.Key, kvp.Value);
                 }
 
-                var result = new PSObject[0];
+                PowerShellResult result = null;
                 try
                 {
-                    result = powerShell.Invoke()?.ToArray();
+                    PSObject[] output = powerShell.Invoke()?.ToArray();
+                    ErrorRecord[] errors = powerShell.Streams?.Error?.ToArray();
```

Comment:
Might be able to drop the null conditionals, as far as I know `Streams` and `Error` are always populated.

Come to think of it, I don't think `powerShell.Invoke()` can return `null` either, just an empty collection.